### PR TITLE
Gift Entry: payment services updates

### PR DESCRIPTION
**Pull request by psadie-sfdo**
[W-9419784](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07AH000000O5rSYAS/view)

Update Gift Entry UI and Elevate payments behavior based on if the org is Elevate registered or not.

# Critical Changes

# Changes

# Issues Closed

# Community Ideas Delivered

# Features Intended for Future Release

# Features for Elevate Customers

# New Metadata
Custom Labels:

- geElevateWidgetPaymentServiceUnavailable
- gePaymentServicesUnavailableHeader
- gePaymentServicesUnavailableBody
- gePaymentServicesUnavailableFailureMessage

# Deleted Metadata

### CODE REVIEW

File: force-app/main/default/classes/Gifts.cls [hunk 1]
```diff
@@ -93,6 +95,25 @@ public with sharing class Gifts {
...
 
+    @TestVisible
+    private void giftsToProcessingStatus() {
+        Boolean paymentIntegrationUnavailable =
+            !paymentIntegrationServiceConfig.isIntegrationEnabled();
+
+        for (DataImport__c gift : dataImports) {
+            Gift currentGift = new Gift(gift);
+
+            if (currentGift.hasAuthorizedPayment() && paymentIntegrationUnavailable) {
```
Comment (kenlewis-sfdo) [inline force-app/main/default/classes/Gifts.cls:106]: No change needed right now, but this is an example where a GiftPayments domain would be helpful.  currentGift.hasAuthorizedPayment() should be Payment.isAuthorized() (or Payments.authorized()).  GiftPayments would be a collection of Payments that know their GiftId.

File: force-app/main/default/classes/Gifts.cls [hunk 2]
```diff
@@ -54,21 +65,12 @@ public with sharing class Gifts {
...
-    public void giftsToProcessingStatus() {
+    public void prepareGiftsForProcessing() {
         giftsToUpdate.clear();
-        for (DataImport__c gift : dataImports) {
-            if (isProcessableGift(gift)) {
-                gift.Status__c = BDI_DataImport_API.bdiProcessing;
-                giftsToUpdate.add(gift);
-            }
-        }
+        giftsToProcessingStatus();
```
Comment (kenlewis-sfdo) [inline force-app/main/default/classes/Gifts.cls:70]: Right now this method does two things - I'd like to see the method split and both be expressed here at the call site for clarity - if(hasPmts && serviceUnavail()) {failGiftsWithPaymentsWithReason('reason'); } else {giftsToProcessingStatus();}

**Review by kenlewis-sfdo — COMMENTED** [review]
A few comments but this is looking really good and the tests look great!

**Review by kenlewis-sfdo — APPROVED** [review]
This is great work, Prince!

**Review by andrewyu-salesforce — APPROVED** [review]
Great work

**Comment by melissabarber** [comment]
@psadie-sfdo Reviewed the labels and added one small tweak to the shortDesc of `geElevateWidgetPaymentServiceUnavailable`. Other than that, all the labels look good!

**Pull request merged**